### PR TITLE
Docs

### PR DESCRIPTION
**Pull request by ABohynDOE**
Adding a the build/ and source/ fodler from the sphinx documentation inside the docs folder. 
The docs/source/index.html can later be used to create a github page

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -134,12 +134,7 @@ dmypy.json
 # Pyre type checker
 .pyre/
 
-# Developers
```
Comment (dmbarreiro) [inline .gitignore:137]: I don't think these lines should be removed

File: utils/validation/keywords.yaml [hunk 2]
```diff
@@ -7,4 +7,8 @@ keywords:
   - center-runs
   - fractional-factorial
   - plackett-burman
+<<<<<<< HEAD
```
Comment (dmbarreiro) [inline utils/validation/keywords.yaml:10]: there is an unresolved merge conflict here

File: utils/validation/scripts/keywords_validation.py [hunk 3]
```diff
@@ -0,0 +1,30 @@
+"""Check if the list of keywords in validation/ matches the one from the glossary."""
```
Comment (dmbarreiro) [inline utils/validation/scripts/keywords_validation.py:1]: I'm not sure this script is needed, utils/validation/scripts/validation.py is doing something similar already and you can now execute it as a separate script with python utils/validation/scripts/validation.py --file experiments/my-experiment.yaml
It checks against keywords present in utils/validation/keywords.yaml only but that should be enough I believe.

File: utils/data_processing/csv2json.py [hunk 4]
```diff
@@ -108,13 +96,17 @@ def main(infile, outfile, header, units, coded, response, title, doi, source,
     # Loading file to a dataframe to keep column name
     df = pd.read_csv(
         # FIXME problem with the encoding for variables containing greek letters
-        infile, header=header_value, index_col=None, encoding="utf-8", sep=";", decimal=decimal
+        infile,
+        header=header_value,
+        index_col=None,
+        encoding="windows-1252",
+        sep=";",
```
Comment (dmbarreiro) [inline utils/data_processing/csv2json.py:103]: Is this adding the changes @saif-analyze needed to convert his experiments?

**Review by dmbarreiro — CHANGES_REQUESTED** [review]
I think there were some conflics during the merge. I believe keywords_validation.py is not needed. Could Saif test again with his files already or you need to adapt csv2json.py further?
Thanks!

File: utils/data_processing/csv2json.py (hunk 4 above)
Comment (ABohynDOE) [reply utils/data_processing/csv2json.py:103]: Not yet, I need to add another option to do this

File: .gitignore (hunk 1 above)
Comment (ABohynDOE) [reply .gitignore:137]: I added them back

File: utils/validation/scripts/keywords_validation.py (hunk 3 above)
Comment (ABohynDOE) [reply utils/validation/scripts/keywords_validation.py:1]: This is more from a documentation POV, to make sure that all keywords that are the `utils/validation/keywords.yaml` list are indeed explained and defined in the documentation. 
But if it's not needed we can remove it later on.

Comment (dmbarreiro) [reply utils/validation/scripts/keywords_validation.py:1]: I understand, maybe we run it in the github actions to make sure the documentation is always in sync before merge. I'll try to do that later and maybe do some renaming of the files in utils/validation/scripts to make them a bit more descriptive

**Review by dmbarreiro — APPROVED** [review]
good job

**Pull request merged**